package.json

```diff
@@ -44,49 +44,50 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build ngx-papaparse --prod",
+    "build": "ng build ngx-papaparse --configuration production",
     "test": "ng test ngx-papaparse --browsers PhantomJS --watch=false",
     "test:coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "papaparse": "^5.3.2",
```

Comment:
Won't this require the user to manually install papaparse?

Reply:
It is a peer dependency now and those are installed automatically with newer npm versions. No manual interation required, same as before. Advantage is that we avoid installation of multiple different versions in sub folders.

Reply:
Ah, i was not aware they made that change to NPM. 👍